### ISSUE TITLE
[Bug]: GDLauncher unable to join world

### ISSUE THREAD

**Issue by ExodiaJedi** [FTBTeam/FTB-Modpack-Issues#1039]
### Modpack

FTB Plexiglass Mountain

### Modpack version

1.1.1

### Log Files

[Plexi_Error.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/8997934/Plexi_Error.log)
[Test2_Error.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/8997935/Test2_Error.log)

### Describe the bug

Fresh install of 1.1.1 from GDLauncher
Started new Single Player game
Named world Plexi
Got to Joining world
Then received error

Copied latest.log to Plexi_Error.log

Read README.txt and saw that 6GB should be set to max
Corrected RAM setting in GDLaucher

Started Plexiglass Mountains again
Generated new world named Test2
Same error (at same point)

In review of both logs it seems the error that throws me out of the world is related to the starter huts in both logs
> Line 4326 in Plexi_Error.log
> Line 7154 in Test2_Error.log

### Steps to reproduce

Start Modpack
Select Single Player
Wait for world to load

### Expected behaviour

Load into world

### Screenshots

![Plexiglass_Error](https://user-images.githubusercontent.com/75554897/176099895-0eda42d5-0264-4855-bb2c-dcec228e342d.png)

**Comment by ExodiaJedi** [comment]
Checked Caspule's GitHUB and found a related bug that was closed

https://github.com/Lythom/capsule/issues/61

Might be related to Forge version

**Comment by Raidobw2** [comment]
I'll check with the capsule author tomorrow, for now I'm not sure what's causing the bug. Possibly it's a path resolution problem due to the Capsule's mod logic, but could also be something else.
You can try the FTB App or rename your instance to not use spaces to see if that changes anything in both cases. If it does, please let me know what worked as it will be helpful

**Comment by ExodiaJedi** [comment]
Removed instance and re-downloaded to create a new instance called PlexiglassMountain.
Made sure RAM set to 6GB before initial launch of new instance.

Same error (starting line 6956)

[Test3_Error.log](https://github.com/FTBTeam/FTB-Modpack-Issues/files/8999580/Test3_Error.log)

**Comment by ExodiaJedi** [comment]
Disabled Caspule mod on my instance and able to launch modpack successfully.
Able to play "adjusted" version of modpack, but hopefully issue can be resolved.

**Comment by Raidobw2** [comment]
I've reported this to the mod author, if you haven't tried the FTB App, it could likely fix it. Looks like GDLauncher uses relative paths which possibly don't currently work with the Capsule mod. Plus using the FTB App supports us, so up to you. The FTB App uses absolute paths, so that's why I think it'll "fix" it, at least temporarily while Capsule is worked on

**Comment by NotOnLand** [comment]
So glad I found this, I've been wracking my brain over this exact issue for hours. Disabling Capsule (also using GDLauncher) fixes the issue for me as well.

**Comment by ExodiaJedi** [comment]
> So glad I found this, I've been wracking my brain over this exact issue for hours. Disabling Capsule (also using GDLauncher) fixes the issue for me as well.

Make sure in GDLauncher for your instance of Plexiglass that in Manage (right-click modpack) "Override Java Memory" is OFF (its on by default) ... it will greatly help performance

**Comment by DrLucky007** [comment]
Hi,

I've experienced the same problem as OP:  GDLauncher, Plexiglass Mountain, versions 1.1.0 and 1.1.1 tested.  Log shows capsule resource missing; Single player world closes with "Invalid Player Data" on new world.

Following the suggestions in thread, I installed PM (v 1.1.1) via FTB App instead.

Created a world, was able to get into the world.

So the bug may be related to GDLauncher.

**tl;dr: Had bug above bug with GDLauncher, unable to replicate in FTB App**

**Comment by NyarlathotepZERO** [comment]
I've had the exact same issue as above and tried numerous things to no avail (I'm also using GDLauncher). I have just tried disabling capsule mod and it definitely worked.

I'm running Plexi v1.2.0
